src/components/FIRAPoints.tsx

```diff
@@ -0,0 +1,67 @@
+import { Table, TableHead, TableRow, TableCell, TableBody, Stack } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { Results } from 'crewtimer-common';
+import React from 'react';
+import { firaPointsCalc } from '../calculators/FIRAPointsCalc';
+
+const StyledTableRow = styled(TableRow)(({ theme }) => ({
+  '&:nth-of-type(odd)': {
+    backgroundColor: theme.palette.action.hover,
+  },
+  // hide last border
+  '&:last-child th': {
+    border: 0,
+  },
+}));
+
+const HeaderTableCell = styled(TableCell)(() => ({
+  fontWeight: 'bold',
+}));
+
+export const FIRAPointsTraditional: React.FC<{ results: Results }> = ({ results }) => {
+  const points = firaPointsCalc(results);
+
+  const maxDataRows = Math.max(points.men.length, points.women.length, points.overall.length);
+  const tableBody: JSX.Element[] = [];
+  for (let row = 0; row < maxDataRows; row++) {
+    tableBody.push(
+      <StyledTableRow key={row}>
+        <TableCell></TableCell>
+        <TableCell align='left'>{row + 1}</TableCell>
+        <TableCell>{points.men[row]?.team || ''}</TableCell>
+        <TableCell>{points.men[row]?.points || ''}</TableCell>
+        <TableCell></TableCell>
```

Comment:
If the intent is to have some 'space' between columns, you can do this by adding `sx={{paddingRight: '2em'}} to the prior cell.  Having empty cells works too of course.

Reply:
I'll play with the visualizer as well after I fix the "tie" situation. I think this was a copy/paste from the code you sent me. What does the '2em' effectively mean? I saw that throughout the React code.